app/src/main/res/values/floors.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string-array name="h_floors_array">
```

Comment:
As I am doing this review, I see on the GUI 1/2 and 4, with 1/2 representing 9th floor and that being above the label for 4th floor. Anyways I think the upper floor should be above a lower floor in the floors menu. When I looked at this yesterday the order was inversed with 4,5,9 meaning 9 was in the bottom.